src/index.ts

```diff
@@ -144,12 +144,17 @@ module.exports = class VendureApi extends VendureApiBase {
      * Adds additional resolve functions to the stitched GQL schema for the sake of data-splitting
      */
     addTypeResolvers() {
-        const resolvers = {
+        const resolvers: IResolvers = {
             Category: {
                 products: (...args: [Category, ProductsCategoryArgs]) => this.categoryProducts(...args),
             },
         };
-        addResolveFunctionsToSchema({ schema: (this.gqlServerConfig as any).schema, resolvers });
+
+        addResolveFunctionsToSchema({
+            schema: this.gqlServerConfig.schema,
+            resolvers,
+            resolverValidationOptions: { allowResolversNotInSchema: true, requireResolversForResolveType: false },
+        });
```

Comment:
For some unknown reason, the past code was crashing when the main page on storefront is opened. It  returns the error `Category was defined in resolvers, but it's not an object`.

I added these flags to ignore the validations, but it isn't the best solution and isn't really fix this error, because know no product is returned on frontend.

Reply:
I just checked out your PR locally with the latest version of Falcon Client (1.3.1). I do not see the "Category was defined..." error, even when I switch off the `resolverValidationOptions`. Also the client storefront seems to work, as in it correctly displays the product list and detail views.